cmd/nomos/status/client.go

```diff
@@ -354,6 +358,18 @@ func (c *ClusterClient) namespaceRepoClusterStatus(ctx context.Context, cs *Clus
 	}
 }
 
+// IsOssInstallation will check for the existence of ConfigManagement object, Operator deployment, and root-reconciler deployment
+// If root-reconciler exist but ConfigManagement and Operator doesn't, it indicates an OSS installation
+func (c *ClusterClient) IsOssInstallation(ctx context.Context) bool {
+	v, _ := c.ConfigManagement.Version(ctx)
+	_, operatorDeployment := c.K8sClient.AppsV1().Deployments(configmanagement.ControllerNamespace).Get(ctx, ACMOperatorDeployment, metav1.GetOptions{})
+	_, rootDeployment := c.K8sClient.AppsV1().Deployments(configmanagement.ControllerNamespace).Get(ctx, "root-reconciler", metav1.GetOptions{})
+	if operatorDeployment != nil && v == util.NotInstalledMsg && rootDeployment == nil {
```

Comment:
`operatorDeployment != nil`: it returns true only when it is not-found error. For other types of error, like connection error, we would need to handle it differently.

Reply:
updated to check that the error that we want for oss is NotFound error.